### DISCUSSION TITLE
Error while using Truffle unbox react

### DISCUSSION THREAD

**Discussion by samcms1234** [ConsenSys-archive/truffle#5867] [Truffle Boxes]
**Issue**
The issue is when I use Truffle unbox react it throws an error

**Steps to Reproduce**
truffle unbox react

**Expected Behavior**
Command should run properly without any error messages.

**Actual Results**
`truffle unbox react python2
Traceback (most recent call last):
  File "/home/divyanshu/.local/bin/truffle", line 5, in <module>
    from truffle.app import main
  File "/home/divyanshu/.local/lib/python3.8/site-packages/truffle/app.py", line 33
    print file_index
          ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(file_index)?`

**Environment**
- Operating System: Ubuntu 20.04.5 LTS
- Ethereum client:
- Truffle version (truffle version):Truffle v5.7.4
- node version (node --version): v16.18.1
- npm version (npm --version): 8.19.3

**Comment by cipaterson** [comment]
Hi @samcms1234,
I can't replicate your error. The error 

> print file_index ^ SyntaxError: Missing parentheses in call to 'print'. Did you mean print(file_index)?

Indicates that python 3 is trying to run code written for python 2 (requiring parentheses on print is one of the syntax changes from python 2 to 3), however  I'm unsure why a truffle unbox would involve python at all.

I don't have this path:
.../lib/python3.8/site-packages/truffle/app.py on my machine to check it, possibly change it to working python 3 code.

Please let us know if the react truffle box is unboxed ok, regardless of the python error?
